src/sentry/integrations/jira/integration.py

```diff
@@ -360,6 +360,8 @@ def get_group_description(self, group, event, **kwargs):
             )
         ]
 
+        # TODO: We probably need to modify something here so that we can use the occurrence to do
+        # this customization rather than the performance event.
```

Comment:
I think so, as long as we include the info in `evidence_display` as part of the detectors

Reply:
Also I know this is a WIP but just in case, there's custom code in several integrations for performance issues we can probably nuke and have it follow the generic issue paths. I can point those out if you'd like.

Reply:
Yeah if I've missed them in this pr please do, I mostly just searched on `GroupCategory.PERFORMANCE`

Reply:
[GitHub](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/issues.py#L54) (and the associated function)

[mixins](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/mixins/issues.py#L97)

special casing for slack info level [here](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/issues.py#L224-L227) and fallback text [here](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/base/base.py#L43-L44)

some special casing for email templates [here](https://github.com/getsentry/sentry/blob/master/src/sentry/eventstore/models.py#L798)

This debug view won't need to exist anymore: https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/debug/debug_performance_issue.py

We won't need the emails - [html](https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/performance.html), [txt](https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/performance.txt), [fixture](https://github.com/getsentry/sentry/blob/master/fixtures/emails/performance.txt)

I think that's all 😅 I had written stuff for a few other integrations but thankfully it uses shared functions.

Reply:
Agh maybe ignore the comments about emails, just read the stuff below.

Reply:
Thanks, I've update the ones that I missed. I think some things here are cleanups that I'll remove once perf issues are fully migrated over.